server-plugins/tracker-resources/src/index.ts

```diff
@@ -69,6 +75,83 @@ export async function issueTextPresenter (doc: Doc, control: TriggerControl): Pr
   return issueName
 }
 
+function isTheSamePerson (control: TriggerControl, assignee: Ref<Person>, target: Ref<Account>): boolean {
+  const targetAccount = control.modelDb.getObject(target) as PersonAccount
+  return assignee === targetAccount?.person
+}
+
+const NOTIFICATION_BODY_SIZE = 50
```

Comment:
I do no think we need to strip it here, inbox presenter should be smart enough to render it properly no matter of size.

Reply:
this is not for inbox presenter, this is for native notifications. So the idea behind this in a tracker is the following. When someone put  a comment in issue activity we want to show the start of the message. However the message itself could be really big, it could have headers and formatting, even table. We have no controls for it, but we allow to paste it. And for whole message having it plain text is might be not the best idea, because we loose rich formatting. But for native notifications we need just plain text.

In this case I think it's just too much to preserve whole message inside a inbox item. So I propose to cut it here for activity messages.

Reply:
I'm not sure why we have to distinguish them, better to have one solid InboxItem and have a notification based on it shown to user, or email was send/etc.